tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
+    "sourceMap": true,
+    "outDir": "lib/",
+    "declaration": true
```

Comment:
I got another PR already lined up based on this one and I can look into adding it there. I wouldn't like to generate too many unnecessary merge conflicts for myself if you're okay with that 😉